src/test/scala/uk/gov/nationalarchives/LambdaTest.scala

```diff
@@ -178,8 +184,48 @@ class LambdaTest extends AnyFlatSpec with BeforeAndAfterEach {
     sfnRequest.stateMachineArn should equal("arn:aws:states:eu-west-2:123456789:stateMachine:StateMachineName")
     sfnRequest.name should equal(s"TEST-REFERENCE-${uuids(4)}")
 
-    input.series should equal("TEST SERIES")
-    input.department should equal("TEST")
+    input.series.get should equal("TEST SERIES")
+    input.department.get should equal("TEST")
+    input.batchId should equal("TEST-REFERENCE")
+    input.s3Prefix should equal("TEST-REFERENCE/")
+    input.s3Bucket should equal(testOutputBucket)
+  }
+
+  "the lambda" should "start the state machine execution with a null department and series if the cite is missing" in {
+    val inputJson =
+      s"""{"parameters":{"TRE":{"reference":"$reference","payload":{"filename":"Test.docx","sha256":"abcde"}},"PARSER":{"uri":"https://example.com","court":"test","date":"2023-07-26","name":"test"}}}"""
+    stubAWSRequests(inputBucket, metadataJsonOpt = Option(inputJson))
+    IngestParserTest().handleRequest(event, null)
+
+    val sfnEvent = sfnServer.getAllServeEvents.asScala.head
```

Comment:
Could some of this go into a private method for all tests to call?

Reply:
Yeah, I've moved some of it out to a private method.